benches/microbenches.rs

```diff
@@ -125,86 +125,6 @@ fn read_namespaced_event(c: &mut Criterion) {
     group.finish();
 }
 
-/// Benchmarks the `BytesText::unescaped()` method (includes time of `read_event`
-/// benchmark)
-fn bytes_text_unescaped(c: &mut Criterion) {
```

Comment:
@Mingun Now that we have benchmarks for unescape() directly, this benchmark is a little redundant, do you think it still provides enough value to keep?

(it's not a new bench, I just relocated it within the file)

Reply:
> this benchmark is a little redundant, 

Agree, can be removed